docs/best-practices.md

```diff
@@ -0,0 +1,119 @@
+# Best practices and tips
+
+This page is about to give contributors a quick overview of tips and best practices for 
+writing in asciidoc. It is for sure not complete nor covers all possibilities, but it's
+intention is to quickly speed up contributors. For a complete reference see the
+[Asciidoctor Documentation](https://asciidoctor.org/docs/).
+
+**Table of Contents**
```

Comment:
A simpler, more efficient approach, is to let Antora/AsciiDoc [generate a table of contents for you](https://asciidoctor.org/docs/user-manual/#user-toc).

Reply:
Question:
this part will not go into the published documentation but will stay in this repository as part of the readme.md, therefore it is markup and not asciidoc - or ?
I would be happy if github renders asciidoc properly for exact that reason you mentioned.

Reply:
Github is rendering asciidoc :)

Reply:
great, will :toc:

Reply:
toc'ed, updated and squashed

Reply:
Thanks for pointing out that it's a Markup file, not AsciiDoc. Not sure why I didn't pick that up when I added my feedback.

Reply:
I heard that GitHub had deprecated AsciiDoc rendering support, @tboerger.

Reply:
![bildschirmfoto 2018-11-26 um 09 02 40](https://user-images.githubusercontent.com/196801/49000504-15ce6a80-f15a-11e8-9b7b-137acb5f5535.png)

Reply:
Going to have a quick look at how this renders. I'm not sure if this will work in this context, but it's an excellent step forward.

Reply:
I converted the file to AsciiDoc and took a look at how it renders in the LivePreview plugin and on GitHub. Unfortunately, the table of contents doesn't render in either. You can see an example of what they look like in the screenshots below.